static/js/admin.js

```diff
@@ -155,16 +155,19 @@ function autoGrade() {
     }
 
     if (assignment != null && question === null && studentID == null) {
+        let i = 0;
         for (let s in students) {
```

Comment:
Is `students` an array of student names? (Aargh, the grading interface doesn't user userids!) I assume there's no easy way around this for now?

Reply:
These are usernames/sids